src/core/tests/style/test_pack.py

```diff
@@ -5,9 +5,9 @@
 from travertino.node import Node
 from travertino.size import at_least
 
+from toga.resources.colors import rgb
 from toga.style.pack import *
```

Comment:
At column 1: (F403) 'from toga.style.pack import *' used; unable to detect undefined names